tests/MultiplexingDistributedCache.Tests/MultiplexingDistributedCacheTests.cs

```diff
@@ -0,0 +1,52 @@
+using Xunit;
+using Aqovia.Cache;
+using Microsoft.Extensions.Caching.Distributed;
+using Microsoft.Extensions.Caching.Memory;
+using Microsoft.Extensions.Options;
+using Moq;
+using System;
+
+namespace Aqovia.Cache.Tests;
+
+public class MultiplexingDistributedCacheTests
+{
+    [Fact]
+    public void Primary_Get_Is_Called_Once()

```

Comment:
I think we should explicitly test that Secondary Get is not called when a secondary cache is defined. (Same for any other operations that secondary cache should not be involved in.)